File: Laba6/Laba6/Models/Task3Class.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Laba6.Models
{
    public class Task3Class : CustomData
    {
        public int appliedForce;


        public Task3Class(int day, int month, int year, int appliedForce) : base(day, month, year)
        {
            this.appliedForce = appliedForce;
        }

        public double CalculateWork(double force, double distance)
        {
            double work = (force * distance) * Math.Cos(0) * appliedForce;

            return work;
        }
    }
}

File: Laba6-Python/TaskModels/task3Class.py
from TaskModels.customData import CustomData


class Task3Class(CustomData):
    def __init__(self, day, month, year):
        super().__init__(day, month, year)
        self.AgeOfProduct = 0

    def calculate_age_of_product(self, manufacturing_year):
        self.AgeOfProduct = manufacturing_year - self.Year
        
    def __str__(self):
        return str(self.Day) + "." + str(self.Month) + "." + str(self.Year)
